types/react-loadable/index.d.ts

```diff
@@ -1,184 +1,190 @@
 // Type definitions for react-loadable 5.3
 // Project: https://github.com/thejameskyle/react-loadable#readme
-// Definitions by: Diogo Franco <https://github.com/Kovensky>, Oden S. <https://github.com/odensc>
+// Definitions by: Diogo Franco <https://github.com/Kovensky>, Oden S. <https://github.com/odensc>, Ian Ker-Seymer <https://github.com/ianks>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.4
 
-import * as React from 'react';
-
-export interface LoadingComponentProps {
-    isLoading: boolean;
-    pastDelay: boolean;
-    timedOut: boolean;
-    error: any;
-}
-
-export type Options<Props, Exports extends object> = OptionsWithoutRender<Props> | OptionsWithRender<Props, Exports>;
-
-export interface CommonOptions {
-    /**
-     * React component displayed after delay until loader() succeeds. Also responsible for displaying errors.
-     *
-     * If you don't want to render anything you can pass a function that returns null
-     * (this is considered a valid React component).
-     */
-    // NOTE: () => null is only needed until React.SFC supports components returning null
-    loading: React.ComponentType<LoadingComponentProps> | (() => null);
-    /**
-     * Defaults to 200, in milliseconds.
-     *
-     * Only show the loading component if the loader() has taken this long to succeed or error.
-     */
-    delay?: number | false | null;
-    /**
-     * Disabled by default.
-     *
-     * After the specified time in milliseconds passes, the component's `timedOut` prop will be set to true.
-     */
-    timeout?: number | false | null;
-
-    /**
-     * Optional array of module paths that `Loadable.Capture`'s `report` function will be applied on during
-     * server-side rendering. This helps the server know which modules were imported/used during SSR.
-     * ```ts
-     * Loadable({
-     *   loader: () => import('./my-component'),
-     *   modules: ['./my-component'],
-     * });
-     * ```
-     */
-    modules?: string[];
-
-    /**
-     * An optional function which returns an array of Webpack module ids which you can get
-     * with require.resolveWeak. This is used by the client (inside `Loadable.preloadReady`) to
-     * guarantee each webpack module is preloaded before the first client render.
-     * ```ts
-     * Loadable({
-     *  loader: () => import('./Foo'),
-     *  webpack: () => [require.resolveWeak('./Foo')],
-     * });
-     * ```
-     */
-    webpack?: () => number[];
+/// <reference types="react" />
+
+declare namespace Loadable {
```

Comment:
This creates a global `Loadable` identifier, though.

Should be moved inside the `declare module` block.